src/RulesEngine/RulesEngine/NullLogger.cs

```diff
@@ -1,18 +1,27 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
 using System;
```

Comment:
Can we remove this as well and replace with - https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.abstractions.nulllogger?view=dotnet-plat-ext-3.1 ?

Reply:
Yes - I like that as well.

Reply:
@abbasc52 I have made the updates using ```NullLogger<RulesEngine>``` instead.  Let me know if you find anything else and when/if you can release a new version.

Reply:
Thanks. We are actually looking at redefining our namespace so we will be taking that up and release version 2 of the nuget. We are planning to release the new version by next week.